src/components/displayplants.js

```diff
@@ -1,19 +1,30 @@
 import React from "react"
+import hash from 'object-hash'
 
 const DisplayPlants = ({ plants, plantImage }) => {
   return (
-    plants.map((plant, index) => {
-      if ((index === 20) && plants.length > 25) { return <span className="h4"> ...and {plants.length - 20} more!</span> }
-      if ((index > 19) && plants.length > 25) { return null }
-      return <img src={plantImage}
-        title={"Name: " + plant.plant_name + "\n" +
-          "County: " + plant.county + "\n" +
-          "Megawatt Capacity: " + plant.capacity_mw + "\n" +
-          "Utility: " + plant.utility_name + "\n"}
-        alt="a power plant which needs to be replaced"
-      >
-      </img>
-    })
+    <div className="row">
+      { plants.map((plant, index) => {
+        if ((index === 10) && plants.length > 15) { return <span key={hash(plant)} className="h4"> ...and {plants.length - 10} more</span> }
```

Comment:
Can we actually make these numbers constants so we can easily tweak this? Otherwise these are a real case of magic numbers.